src/Assets/AssetDataRegistry.php

```diff
@@ -76,6 +76,7 @@ protected function get_core_data() {
 		global $wp_locale;
 		$currency = get_woocommerce_currency();
 		return [
+			'wp_version'    => get_bloginfo( 'version' ),
```

Comment:
All other settings use camel case, should we update this one to be `wpVersion`?

Reply:
Yup, let's keep things consistent 👍 (see 5560d43)